pages/common/vimdiff.md

```diff
@@ -0,0 +1,24 @@
+# vimdiff
+
+> Open up to four files in Vim and show their differences.
+> See Vim documentation for how to edit and navigate within a window.
+
+- Open files and show the differences:
+
+`vimdiff {{file1}} {{file2}}`
+
+- Use horizontal window split instead of the default vertical split:
```

Comment:
This is just for the 2-file case, right? If so, maybe we should mention it (e.g. "Open two files with a horizontal...")